sem-semantic-release

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SEMANTIC_RELEASE_PLUGINS=()
+SEMANTIC_RELEASE_OPTIONS=""
+SEMANTIC_RELEASE_VERSION=""
+
+semantic-release::parse_args() {
+    local PARSING_PLUGINS=1
+    local PARSING_BRANCHES=1
+    local DRY_RUN=1
+
+    local SEMANTIC_RELEASE_BRANCHES=()
+
+    set -f
+
+    while [[ $# -gt 0 ]]; do
```

Comment:
I'm not sure about this, but I know something that can help.
Please add sem-semantic-relase to the Bash linter block in the Semaphore YAML.